runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -322,16 +330,17 @@ void BaseLoader<LoaderDomain, SpecificLoader>::loadOperationIO(const Operator *o
   {
     // Optional tensors are not supported yet except for FULLY_CONNECTED.
     auto builtin_code = _model->operator_codes()->Get(op->opcode_index())->builtin_code();
-    if (isOptionalInputTensor(idx))
+    if (isOptionalInputTensor(idx) &&
+        builtin_code != BuiltinOperator::BuiltinOperator_FULLY_CONNECTED)
```

Comment:
Could the condition check for `builtin_code` be expanded to include also other ops that allow for optional input tensor? A list of allowed ops, or a flag for each op indicating use of optional input tensors would help greatly. In future, developers can simply add their `builtin_code` to allow optional input tensors. Do you think this is feasible?

Reply:
@dr-venkman Thank you for suggestion. Updated.